clustermesh/clustermesh.go

```diff
@@ -1885,3 +1939,224 @@ func DisableWithHelm(ctx context.Context, k8sClient *k8s.Client, params Paramete
 	_, err = helm.Upgrade(ctx, k8sClient.RESTClientGetter, upgradeParams)
 	return err
 }
+
+func getRelease(kc *k8s.Client, namespace string) (*release.Release, error) {
+	client := kc.RESTClientGetter
+	release, err := helm.GetCurrentRelease(client, namespace, defaults.HelmReleaseName)
+	if err != nil {
+		return nil, err
+	}
+	return release, nil
+}
+
+// validateCAMatch determines if the certificate authority certificate being
+// used by aiLocal and aiRemote uses a matching keypair. The bool return value
+// is one of: nil (unknown), true (keypairs match), false (keys do not match)
+func (k *K8sClusterMesh) validateCAMatch(aiLocal, aiRemote *accessInformation) (*bool, error) {
```

Comment:
Nit. I would suggest to return a plain bool rather than a pointer (for the sake of simplicity). It currently seems to be nil only when an error is not nil.

Reply:
I'll make this change, but it feels wrong to me to not return "maybe" somehow. I agree that this is a simple enough function that it's unlikely to cause any issues.